src/Microsoft.PowerShell.Commands.Utility/commands/utility/select-object.cs

```diff
@@ -334,9 +334,11 @@ private void ProcessExpressionParameter()
                 _expandMshParameterList = processor.ProcessParameters(new string[] { ExpandProperty }, invocationContext);
             }
 
-            if (ExcludeProperty != null)
+            // ExpandProperty skip processing ExcludeProperty
+            if (ExcludeProperty != null && string.IsNullOrEmpty(ExpandProperty))
```

Comment:
We cannot ignore `-ExcludeProperty` just because `-ExpandProperty` is specified. See the script below:
```
PS:88> $s = Get-Process -Id $pid | Select-Object -Property Process* -ExpandProperty Modules
PS:89> $s[0].ProcessName
powershell
PS:90> $s[0].ProcessorAffinity
255
PS:91> $p = Get-Process -Id $pid | Select-Object -Property Process* -ExcludeProperty ProcessorAffinity
-ExpandProperty Modules
PS:92> $p[0].ProcessName
powershell
PS:93> $p[0].ProcessorAffinity
PS:94>
```
`-ExcludeProperty` took effect and excluded the property `ProcessorAffinity`, so only `ProcessName` got added to each Module object.

I feel it's OK to break the `Select-Object -ExcludeProperty value1 -ExpandProperty value2` usage based on my analysis of the usage of `Select-Object` in GitHub. Please see my comment above.